internal/resources/slo/data_source_slo.go

```diff
@@ -99,6 +100,22 @@ func unpackQuery(apiquery gapi.Query) []map[string]interface{} {
 		retQuery = append(retQuery, query)
 	}
 
+	if apiquery.Type == gapi.QueryTypeRatio {
```

Comment:
nit - it'd be more idomatic in golang to use a `switch apiQuery.Type {` with two case statements (and a default if we know how to handle the case where it didn't match either). You could use `if {} else if {}` as well, but the switch is even clearer to reader that only one of the blocks is going to run